typescript/sdk/src/timelock/evm/EvmTimelockDeployer.ts

```diff
@@ -0,0 +1,65 @@
+import { ethers } from 'ethers';
+import { Logger } from 'pino';
+
+import { TimelockController__factory } from '@hyperlane-xyz/core';
+import { rootLogger } from '@hyperlane-xyz/utils';
+
+import { HyperlaneDeployer } from '../../deploy/HyperlaneDeployer.js';
+import { ContractVerifier } from '../../deploy/verify/ContractVerifier.js';
+import { HyperlaneContracts } from '../../index.js';
+import { MultiProvider } from '../../providers/MultiProvider.js';
+import { TimelockConfig } from '../types.js';
+
+export class EvmTimelockDeployer extends HyperlaneDeployer<
+  TimelockConfig,
+  {
```

Comment:
would prefer if this was broken out into a separate type a la https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/eace222669f27c4c5328686a04bdc3f00e9f7464/typescript/sdk/src/hook/contracts.ts#L19